src/tests/ftest/server/metadata_svc_ops.py

```diff
@@ -0,0 +1,143 @@
+"""
+
+import statistics
+import time
+
+from apricot import TestWithServers
+from avocado.core.exceptions import TestFail
+from general_utils import DaosTestError
+from thread_manager import ThreadManager
+
+
+class DuplicateRpcDetection(TestWithServers):
+    """Compare metadata handling performance between pools with and without duplicate rpc
+       detection feature enabled.
+
+    Test Class Description:
+        Create pools run test with and without duplicate rpc detection feature and verify time
+        consuming with metadata workload before and after svc ops full.
+
+    :avocado: recursive
+    """
+
+    def setUp(self):
+        """Set Up DuplicateRpcDetectionTest"""
+        super().setUp()
+        self.dmg = self.get_dmg_command()
+
+    def metadata_workload_test(self, pool=None, num_of_cont=1, workload_cycles=5000, test_loops=1):
+        """To create single container and perform metadata workload tests.
+
+        Args:
+            pool (str): pool handle to create container. Defaults to None.
+            num_of_cont (int): Number of container to be created. Defaults to 1.
+            workload_cycles (int): Number of metadata workload test cycles per test loop.
+                Defaults to 5000.
+            test_loops (int): Number of metadata workload test loops. Defaults to 1.
+
+        Returns:
+            test_time: List of time consumed per test loops of metadata workload test cycles.
+
+        """
+        test_time = []
+        try:
+            container = self.get_container(pool, create=False)
+            if container.daos:
+                container.daos.verbose = False
+            container.create()
+            self.log.info("Successfully created #%s container", num_of_cont)
+        except (DaosTestError, TestFail) as err:
+            self.fail(
+                "#({}.{}) container create failed. err={}".format(pool.label, num_of_cont, err))
+        for ind in range(test_loops):
+            start = time.time()
+            for _ in range(workload_cycles):
+                container.close()
+                container.open()
+            elapsed_time = time.time() - start
+            self.log.info("Completed container Metadata test-loop: %d, elapsed_time: %f",
+                          ind + 1, elapsed_time)
+            test_time.append(elapsed_time)
+        for ind in range(test_loops):
+            self.log.info("Test time of Metadata test-loop: %d,  %f",
+                          ind, test_time[ind])
+        container.close()
+        return test_time
+
+    def test_metadata_dup_rpc(self):
+        """JIRA ID: DAOS-15937 metadata duplicate rpc detection time consuming.
+
+        Test Steps:
+            1. Bring up DAOS server.
+            2. Create pool1 with specified property svc_ops_entry_age.
+            3. Create containers by ThreadManager.
+            4. Run specified metadata workload cycles in multiple test loops (N cycles per loop),
```

Comment:
```suggestion
            4. Run specified metadata workload cycles in multiple test loops (N cycles per loop).
```

Reply:
and suggest removing the next 2 lines of text, because the average time is not calculated.

Reply:
Thanks for the comment. 
Please see the new commit.